docs/encrypted_images.md

```diff
@@ -147,6 +147,13 @@ the status area just before starting the upgrade process, because it
 would be very hard to determine this information when an interruption
 occurs and the information is spread across multiple areas.
 
+## [Factory-programing requirement:](#factory-programing-requirement)
+
+It is important to have updates without any voids in encryption, therefore
+from the very begining, flags and TLV's has to set accordingly.
+Perform first flashing with encryption settings as if it was meant for DFU
+and add `--clear` flag to keep image unencrypted and redy to go.
```

Comment:
Perform the first flashing with image signed by imgtool with encryption settings (...) appended by '--clear` (..)